modules/logging-loki-pod-placement.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+// cluster-logging-loki.adoc
```

Comment:
Unless this is a feature-specific practice, we usually include the full path to the assembly to make it easier.
 ```suggestion
// logging/log_storage/cluster-logging-loki.adoc
```